pom.xml

```diff
@@ -130,6 +130,14 @@
         <jetty.version>9.4.28.v20200408</jetty.version>
         <git-commit-id-plugin.version>2.2.6</git-commit-id-plugin.version>
         <scala.version>2.13.2</scala.version>
+
+        <netty-tcnative-version>2.0.39.Final</netty-tcnative-version>
+        <!-- We normally get this from common, but Vertx is built against 4.1.49.Final -->
+        <netty.version>4.1.60.Final</netty.version>
+        <!-- This matches the version in common -->
+        <netty-codec-http2-version>4.1.49.Final</netty-codec-http2-version>
+        <!-- Brought in by both schema registry and connect, we need to pick a version -->
+        <jersey-common>2.31</jersey-common>
```

Comment:
The fixed version is 2.34 though

Reply:
thanks for catching this, not sure where my copy-pasta went wrong. looks like the above is also not right (hsoudl be .60.Final)

Reply:
Yep https://github.com/confluentinc/ksql/blob/6.2.x/pom.xml#L144
I don't see jersey defined in 6.2.x though.

Reply:
I took it from https://github.com/confluentinc/ksql/commit/2425f163134a823871d6756c23d39e795f117d6c